en/docs/tutorials/scenarios/scenario5-developer-community-feature.md

```diff
@@ -1,4 +1,47 @@
-# Scenario 5 - Developer Community Feature
+# Scenario 5 - Getting the developer community involved
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**Quantis allows random users to access their apis. Since they have now open the dev portal for external users to register to their system,  they now expect the developer community to interact with them on building their own applications using their services. Quantis wants to provide a place where app developers can communicate with the Quantis API developers with their API related queries.  This way application developers and external users can understand more about Quantis APIs and they could have better developer experience.**
```

Comment:
```suggestion
Quantis allows random users to access their APIs. Since they have now opened the Developer Portal for external users to register to their system, they now expect the developer community to interact with them on building their own applications using their services. Quantis wants to provide a place where application developers can communicate with the Quantis API developers with their API related queries. This way application developers and external users can understand more about Quantis APIs and they could have a better developer experience.
```

Reply:
Fixed